AddSevenPS/Library.fs

```diff
@@ -17,10 +17,12 @@ module AddSevenPS=
         [<Parameter(Position = 0)>]
         member val Num : string = "" with get , set
         override this.EndProcessing() =
+                 
+            let mutable result = 0
```

Comment:
@Aryan-sharma11 , notice the use of mutable variable here and how this is then used in Line-23 (via `&result`) to store the parsed number upon success.

There are better ways to manage this, such as exception and active-patterns - but I leave that task of refactoring this to you when you have gone through the F# codebases and manuals and have an "expert" opinion on how this should be done.